gateway-outgoing.yaml

```diff
@@ -150,6 +150,10 @@ components:
             independent results.
           items:
             $ref: '#/components/schemas/AssessmentResult'
+        status:
```

Comment:
nit:  Next three lines are indented extra bigly compared to others at same YAML depth.